static/app/views/issueDetails/groupOpenPeriods.tsx

```diff
@@ -0,0 +1,127 @@
+import styled from '@emotion/styled';
+import moment from 'moment-timezone';
+
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  type GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import LoadingError from 'sentry/components/loadingError';
+import {t} from 'sentry/locale';
+import {ISSUE_PROPERTY_FIELDS} from 'sentry/utils/fields';
+import {useParams} from 'sentry/utils/useParams';
+import {useGroup} from 'sentry/views/issueDetails/useGroup';
+
+export const ALL_EVENTS_EXCLUDED_TAGS = [
+  'environment',
+  'performance.issue_ids',
+  'transaction.op',
+  'transaction.status',
+  ...ISSUE_PROPERTY_FIELDS,
+];
+
+type OpenPeriodDisplayData = {
+  duration: string;
+  end: string;
+  start: string;
+  title: string;
+};
+
+// TODO(snigdha): make this work for the old UI
+// TODO(snigdha): suppot pagination
+function IssueOpenPeriodsList() {
+  const params = useParams<{groupId: string}>();
+  const {
+    data: group,
+    isPending: isGroupPending,
+    isError: isGroupError,
+    refetch: refetchGroup,
+  } = useGroup({groupId: params.groupId});
+
+  if (isGroupError) {
+    return <LoadingError onRetry={refetchGroup} />;
+  }
+
+  // update the open periods to have date objects
+  const openPeriods = group?.openPeriods?.map(period => ({
+    ...period,
+    start: new Date(period.start),
+    end: period.end ? new Date(period.end) : null,
+  }));
+
+  const now = new Date();
```

Comment:
needs to be in state at the top or it will change every render since the current time keeps changing
```suggestion
  const [now] = useState(() => new Date())
```